conmon-rs/server/src/container_log.rs

```diff
@@ -12,47 +12,56 @@ pub type SharedContainerLog = Arc<RwLock<ContainerLog>>;
 pub struct ContainerLog {
     drivers: Vec<LogDriver>,
 }
-
 #[derive(Debug)]
 enum LogDriver {
     ContainerRuntimeInterface(CriLogger),
+    Json(JsonLogger),
 }
 
 impl ContainerLog {
-    /// Create a new default SharedContainerLog.
+    //Create a new default SharedContainerLog
```

Comment:
Those should be doc comments, please revert your change.
```suggestion
    /// Create a new default SharedContainerLog
```

Reply:
ok